index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device0width">
+  <title></title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><a>the offer</a></li>
+        <li><a>portfolio</a></li>
+        <li><a>about us</a></li>
+        <li><a>contact</a></li>
+      </ul>
+    </nav>
+    <section>
+      <h2>hello!</h2>
+      <p>
+        We are creative agency based in Warsaw, Poland <br>
+        Creating projects for clients all over the world
+      </p>
+    </section>
+    <section>
+      <h2>design!</h2>
+      <p>
+        Good design is our domain <br>
+        We work in multidisciplinary teams.
+      </p>
+    </section>
+    <section>
+      <h2>projects!</h2>
+      <p>
+        We always select varied projects, <br>
+        to step out of comfort zone.
+      </p>
+    </section>
+  </header>
+
+  <main>
+    <p>scroll down to find out more about our studio</p>
+    <article>
+      <h2>works</h2>
+      <article class="left-aside">
+        <article>
+          <h3>pineapp.</h3>
+          <p>
+            Period: March-April 2016 <br>
+            Team: John (UI), Edwin (UX) <br>
+            Jessica (Front-end)
+          </p>
+          <a href="">visit the website</a>
+        </article>
+        <aside>
+          <img src="pineapple.jpg">
+          <aside>branding & web</aside>
+        </aside>
+      </article>
+      <article class="right-aside">
+        <article>
+          <h3>healthy dining.</h3>
+          <p>
+            Period: March-April 2016 <br>
```

Comment:
You won't need <br>, I believe, you'll have separate spans for a line of texts - also, styling will affect it.